<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextCfger</class>
 <widget class="QFrame" name="TextCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>88</height>
   </rect>
  </property>
  <widget class="QFontComboBox" name="family">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>8</y>
     <width>177</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="familyTag">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>8</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Font:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sizeTag">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>8</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bgColor">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Background Color</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_background_color_64.png</normaloff>:/image/text_background_color_64.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="fgColor">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Foreground Color</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_forground_color_64.png</normaloff>:/image/text_forground_color_64.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="underLine">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Underline</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_underline_64.png</normaloff>:/image/text_underline_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="italic">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Italic</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_italic_64.png</normaloff>:/image/text_italic_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="left">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Align Left</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_left_64.png</normaloff>:/image/text_left_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="center">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Align Center</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_center_64.png</normaloff>:/image/text_center_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="right">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Align Right</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_right_64.png</normaloff>:/image/text_right_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="justify">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Align Justify</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_justify_64.png</normaloff>:/image/text_justify_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bold">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Bold</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_bold_64.png</normaloff>:/image/text_bold_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="wrap">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>48</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Wrap</string>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_wrap_64.png</normaloff>:/image/text_wrap_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="size">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>8</y>
     <width>48</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res/vshow.qrc"/>
 </resources>
 <connections/>
</ui>
